data/lang/module-scout/en.json

```diff
@@ -0,0 +1,482 @@
+  },
+  "EXCELLENT_I_AWAIT_YOUR_REPORT" : {
+    "description" : "",
+    "message" : "Excellent. I will await your report."
+  },
+  "HOW_DOES_IT_WORK" : {
+    "description" : "",
+    "message" : "How does it work."
+  },
+  "INTROTEXT_COMPLETED_1" : {
+    "description" : "",
+    "message" : "Hello! \nIf you are seeing this is because the data were obtained and confirmed what we suspected.\nSee you at:"
+  },
+  "INTROTEXT_COMPLETED_2" : {
+    "description" : "",
+    "message" : "Hello! \nIf you are seeing this is because the data were successfully mapped. Will be very useful to my friends.\n To complete the mission, you must go to:"
+  },
+  "INTROTEXT_COMPLETED_3" : {
+    "description" : "",
+    "message" : "Greetings again. Apparently, the mission has been a success.\nYou must carry this information to:"
+  },
+  "INTROTEXT_COMPLETED_4" : {
+    "description" : "",
+    "message" : "Hello! \nWhat you are seeing this means that the data collected are very interesting and will be very useful to my editor.\n The delivery shall be in:"
+  },
+  "INTROTEXT_COMPLETED_5" : {
+    "description" : "",
+    "message" : "Hello! \nIf you are seeing this is that the mission is successful now. The data will be very useful.\n Your reward awaits you in:"
+  },
+  "INTROTEXT_COMPLETED_6" : {
+    "description" : "",
+    "message" : "You are viewing this because, the first part of mission has been successful.\n To complete it should be directed to:"
+  },
+  "INTROTEXT_COMPLETED_7" : {
+    "description" : "",
+    "message" : "If you are seeing this, the first part of mission seems to have been successful. To complete it you are directed to go to:"
+  },
+  "INTROTEXT_COMPLETED_8" : {
+    "description" : "",
+    "message" : "xxx"
+  },
+  "INTROTEXT_COMPLETED_9" : {
+    "description" : "",
+    "message" : "xxx"
+  },
+  "PLEASE_HAVE_THE_DATA_BACK_BEFORE" : {
+    "description" : "",
+    "message" : "Please have the data back before {date}"
+  },
+  "MAPPING" : {
+    "description" : "As in cartography",
+    "message" : "Mapping"
+  },
+  "MAPPING_INTERRUPTED_ORBITAL" : {
+    "description" : "",
+    "message" : "MAPPING INTERRUPTED. Too far from target. Orbit correction needed."
+  },
+  "MAPPING_INTERRUPTED_SURFACE" : {
+    "description" : "",
+    "message" : "MAPPING INTERRUPTED. Too far from target. Flight ceiling correction needed."
+  },
+  "MAPPING_PAUSED" : {
+    "description" : "",
+    "message" : "MAPPING SUSPENDED. Data incomplete. Further mapping required."
+  },
+  "OBJECTIVE" : {
+    "description" : "",
+    "message" : "Objective:"
+  },
+  "ORBITAL_SCAN" : {
+    "description" : "",
+    "message" : "Orbital Scan"
+  },
+  "SURFACE_SCAN" : {
+    "description" : "",
+    "message" : "Surface Scan"
+  },
+  "AN_ORBITAL_SCAN" : {
+    "description" : "",
+    "message" : "an Orbital scan"
+  },
+  "A_SURFACE_SCAN" : {
+    "description" : "",
+    "message" : "a Surface scan"
```

Comment:
Here as well. "a surface scan"

Reply:
wkfo also found another string set for fixing: "What you are seeing __this__ means that the data collected are very interesting and will be very useful to my editor.⏎ The delivery shall be in:"